src/__tests__/file_collection.subroot.test.ts

```diff
@@ -0,0 +1,187 @@
+import { createReadStream } from 'node:fs';
+import { join } from 'node:path';
+
+import { HttpResponse, http } from 'msw';
+import { setupServer } from 'msw/node';
+import {
+  afterAll,
+  afterEach,
+  beforeAll,
+  beforeEach,
+  describe,
+  expect,
+  it,
+} from 'vitest';
+
+import { FileCollection } from '../FileCollection.ts';
+
+function mapRelativePath(item: { relativePath: string }) {
+  return item.relativePath;
+}
+
+describe.concurrent('basic', () => {
```

Comment:
Not sure why you want this specific suite to be run in parallel? By looking at the inner tests, it doesn't seem useful (nothing is delegated to the libuv thread pool).